app/src/js/common/reducer.ts

```diff
@@ -61,6 +61,10 @@ export const reducer: Reducer<State> = (
       return common.changeViewerConfig(
         state, action as types.ChangeViewerConfigAction
       )
+    case types.SET_MENU_LOCATION:
+      return common.setMenuPosition(
```

Comment:
Better to record the menu position in the react state instead of the global state.